src/netbase.cpp

```diff
@@ -698,23 +698,20 @@ bool LookupSubNet(const std::string& subnet_str, CSubNet& subnet_out)
             uint8_t netmask;
             if (ParseUInt8(netmask_str, &netmask)) {
                 // Valid number; assume CIDR variable-length subnet masking.
-                subnet_out = CSubNet{addr, netmask};
-                return subnet_out.IsValid();
+                return CSubNet{addr, netmask};
```

Comment:
Returning a non-nullopt invalid subnet here and elsewhere is liable to introduce bugs

Reply:
I agree, force-pushed fixing it.